src/dune_rules/pkg_rules.ml

```diff
@@ -337,7 +337,7 @@ module Pkg = struct
      appearing earlier can overwrite the values of variables set by
      packages appearing later. *)
```

Comment:
I think this comment needs to be adapted now because it states the reverse of what is happening and that's very confusing.

Reply:
Good catch. Fixed!